beta/config/main.yml

```diff
@@ -0,0 +1,450 @@
+ansible:
```

Comment:
I don't think you should push this file.

Reply:
whoops. I thought I had added these to my gitignore